apps/portals/nf/src/config/routesConfig.ts

```diff
@@ -186,6 +186,24 @@ const routes: GenericRoute[] = [
       },
     ],
   },
+  {
+    path: 'SubmissionForm1',
+    exact: true,
+    hideRouteFromNavbar: true,
+    synapseConfigArray: [
+      {
+        name: 'DynamicForm',
+        props: {
+          schemaUrl:
+            'https://raw.githubusercontent.com/nf-osi/nf-research-tools-schema/cconrad8-patch-1/NF-Tools-Schemas/SubmitObservationSchema.json',
+          uiSchemaUrl:
+            'https://raw.githubusercontent.com/nf-osi/nf-research-tools-schema/cconrad8-patch-1/NF-Tools-Schemas/SubmitObservationUiSchema.json',
+          postUrl: 'https://submit-form.com/KwZ46H4T',
+        },
+        isOutsideContainer: false,
+      },
+    ],
+  },
   {
```

Comment:
Integrate unlinked page into the NF portal:
<img width="1214" alt="image" src="https://github.com/user-attachments/assets/90eb429e-6178-43e8-a2be-37e9e5b010c0">